src/browser/convert_options.js

```diff
@@ -206,7 +206,6 @@ module.exports = {
         let opts = appJson['startup_app'];
         if (opts) {
             opts.plugin = appJson['plugin'];
-            opts.preload = appJson['preload'];
```

Comment:
defining them on the outside is just wrong and not supported right?

Reply:
was it supported before?

Reply:
Was not supported before; [link to the spec](https://github.com/openfin/Internal-Wiki/wiki/Preload-Scripts)

Reply:
👍

Reply:
:neckbeard: tests surely forthcoming ... merging